code/game/mecha/combat/gorilla.dm

```diff
@@ -0,0 +1,219 @@
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/cannon
+	name = "8.8cm KwK 47"
+	desc = "Do to enemy mechs what the King Tiger did to Allied tanks with 88 milimeters of armor-piercing German steel!"
+	icon_state = "mecha_uac2"
+	equip_cooldown = 55 // 5.5 seconds
+	projectile = /obj/item/projectile/bullet/cannon
+	fire_sound = 'sound/weapons/cannon.ogg'
+	projectiles = 1
+	projectile_energy_cost = 1000
+	salvageable = 0 // We don't want players ripping this off a dead mech.
+
+/obj/item/projectile/bullet/cannon
+	name ="armor-piercing shell"
+	icon = 'icons/obj/grenade.dmi'
+	icon_state = "shell"
+	damage = 1000 // In order to 1-hit any other mech and royally fuck anyone unfortunate enough to get in the way.
+
+	on_hit(var/atom/target, var/blocked = 0)
+		explosion(target, 0, 0, 2, 4)
+		return 1
+
+/* // GLITCHY UND LAGGY.
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/mg42
+	name = "Maschinengewehr 60"
+	icon_state = "mecha_uac2"
+	equip_cooldown = 10
+	projectile = /obj/item/projectile/bullet/midbullet2
+	fire_sound = 'sound/weapons/mg42.ogg'
+	projectiles = 1000
+	projectiles_per_shot = 5
+	deviation = 0.3
+	projectile_energy_cost = 20
+	fire_cooldown = 1
+	salvageable = 0 // We don't want players ripping this off a dead mech.
+*/
+
+/obj/effect/decal/mecha_wreckage/gorilla
+	name = "Gorilla wreckage"
+	desc = "... Blitzkrieg?"
+	icon = 'icons/mecha/mecha64x64.dmi'
+	icon_state = "pzrwreck"
+	layer = 4 // so it overlaps other people
+	pixel_x = -16
+	anchored = 1 // It's fucking huge. You aren't moving it.
+
+/obj/mecha/combat/gorilla
+	name = "Gorilla"
+	desc = "<b>BLITZKRIEEEEEGGGGG!!!</b>"
+	icon = 'icons/mecha/mecha64x64.dmi'
+	icon_state = "pzrmech"
+	initial_icon = "pzrmech"
+	pixel_x = -16
+	step_in = 10
+	health = 5000
+	opacity = 0 // Because there's big tall legs to look through. Also it looks fucky if this is set to 1.
+	deflect_chance = 50
+	damage_absorption = list("brute"=0.1,"fire"=0.7,"bullet"=0.1,"laser"=0.6,"energy"=0.7,"bomb"=0.7) //values show how much damage will pass through, not how much will be absorbed.
+	max_temperature = 25000
```

Comment:
is this magnitudes higher than the temperature of the sun or am i reading wrong

Reply:
_Yes_. I have no idea why I put that there tbh, it's old code. I stole it from the Firefighting Ripley.

Reply:
Sounds like it MIGHT be a good idea to lower it just a bit. If someone gets the station at 5* the temperature of the sun, I think the kill is deserved.